### DISCUSSION TITLE
Chat window monospaced font -> Lyrics/Chords

### DISCUSSION THREAD

**Discussion by JamulusUser** [jamulussoftware/jamulus#1524] [Ideas]
I think it's a shame that html is no longer possible in the chat window. Isn't it possible to introduce at least one non-proportional font (monospaced), so that it is possible to send lyrics with chords above again? 

![prop2](https://user-images.githubusercontent.com/38150953/114847591-24c31f80-9dde-11eb-811f-16f0e05c80b4.png)

**Comment by gilgongo** [comment]
Yes, I believe this was a security thing. It's a pity though, as you say. Maybe that might be refined to allow a subset of tags.

**Comment by JamulusUser** [comment]
I just saw that there is already an identical request:

Sharing lyrics with chords (multiline monospaced messages) in the Chat #1021
https://github.com/jamulussoftware/jamulus/discussions/1021

**Reply by hoffie** [reply]
Yes, let's use #1021 for further discussion. I've tried to post a summary there: https://github.com/jamulussoftware/jamulus/discussions/1021#discussioncomment-666734
